libraries/AP_GPS/AP_GPS_UBLOX.cpp

```diff
@@ -406,6 +422,19 @@ AP_GPS_UBLOX::_request_next_config(void)
         _unconfigured_messages &= ~CONFIG_TIM_TM2;
 #endif
         break;
+
+
+    case STEP_M10: {
+        // special handling of M10 config
+        const config_list *list = config_M10;
+        const uint8_t list_length = ARRAY_SIZE(config_M10);
+        Debug("Sending M10 settings");
+        if (!_configure_config_set(list, list_length, CONFIG_M10, UBX_VALSET_LAYER_RAM | UBX_VALSET_LAYER_BBR)) {
```

Comment:
Why `UBX_VALSET_LAYER_RAM | UBX_VALSET_LAYER_BBR`?

Reply:
because trying to write to the flash layer on the M10 fails